modules/backend/assets/less/layout/sidenav.less

```diff
@@ -83,22 +83,15 @@
     }
 }
 
-@media (min-width: @screen-sm-min) and (max-width: @screen-sm-max) {
-    #layout-sidenav {
-        font-size: 12px;
-    }
-    .layout-sidenav-container {
-        width: 100px;
-    }
+@media (max-width: 769px) {
+	#layout-side-panel {
+		top: 45px;
+	}
 }
-
-@media (max-width: @screen-xs-max) {
-    #layout-sidenav {
-        font-size: 10px;
-    }
-    .layout-sidenav-container {
-        width: 80px;
-    }
+@media (min-width: 769px) {
+	#layout-side-panel {
+		top: 62px;
```

Comment:
@ayumihamsaki, could you also make it `61px`. This white 1px gap drives me crazy 😄

Reply:
@w20k Could you do me a favor pls. Could you tell me your height of the black menu bar at the top of the screen, mine has a height of 62px, see screenshot:

![untitled](https://user-images.githubusercontent.com/17784082/50488233-5dfed800-09f9-11e9-9c02-a48744d6d536.png)

I got a feeling the issue is because October is using `px` and not `rem` for its CSS code.

I just want to double check where the issue is ?

I might just force the top menu to have a height of fixed `62px` instead.

Reply:
See another one of my screenshots:

![untitled](https://user-images.githubusercontent.com/17784082/50488354-f432fe00-09f9-11e9-9124-e626a6c759df.png)

Reply:
Going to change this CSS code:

```css
nav#layout-mainmenu.navbar-mode-inline {
    height: 60px;
}
```

into this:

```css
nav#layout-mainmenu.navbar-mode-inline {
    height: 62px;
}
```

Hopefully that fixes the issue?